src/js/index.js

```diff
@@ -77,13 +88,20 @@ function handleSort() {
 		.forEach(sortedChild => contentEl.append(sortedChild));
 }
 
+function handleExpired() {
```

Comment:
This function is only used in one place 🙂

Reply:
I think we should make handleExpired a part of handleTags because it's a special-case tag